code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -42,16 +42,19 @@
 
 /obj/structure/bed/chair/update_icon()
 	..()
-	if(has_buckled_mobs() && padding_material)
-		var/cache_key = "[base_icon]-armrest-[padding_material.name]"
+	if(has_buckled_mobs())
+		var/cache_key = "[base_icon]-armrest-[padding_material ? padding_material.name : "no_material"]"
 		if(isnull(stool_cache[cache_key]))
 			var/image/I = image(icon, "[base_icon]_armrest")
+			world << "Thought is a nerd."

```

Comment:
I agree with this, but this seems like an oopsie.

Reply:
Fixed 5ever